lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -297,9 +314,7 @@ void FunctionalChaosResult::load(Advocate & adv)
   adv.loadAttribute( "I_", I_ );
   adv.loadAttribute( "alpha_k_", alpha_k_ );
   adv.loadAttribute( "Psi_k_", Psi_k_ );
-  adv.loadAttribute( "composedMetaModel_", composedMetaModel_ );
+  adv.loadAttribute( "isRegression_", isRegression_ );
```

Comment:
we need a backward-compatibility path here

Reply:
Fixed at 1ae9995de68211a961c55a24fda5dc5177e9df52